configuration/configuration.go

```diff
@@ -28,21 +28,22 @@ import (
 
 // Configuration contains configuration params for all Insolar components
 type Configuration struct {
-	Host            HostNetwork
-	Service         ServiceNetwork
-	Ledger          Ledger
-	Log             Log
-	Metrics         Metrics
-	LogicRunner     LogicRunner
-	APIRunner       APIRunner
-	Pulsar          Pulsar
-	VersionManager  VersionManager
-	KeysPath        string
-	CertificatePath string
-	Tracer          Tracer
-	Introspection   Introspection
-	Exporter        Exporter
-	Bus             Bus
+	Host              HostNetwork
+	Service           ServiceNetwork
+	Ledger            Ledger
+	Log               Log
+	Metrics           Metrics
+	LogicRunner       LogicRunner
+	APIRunner         APIRunner
+	APIInternalRunner APIRunner
```

Comment:
rename: APIInternalRunner -> AdminAPIRunner, and in all other places where call it with Internal